config/locales/en.yml

```diff
@@ -60,6 +60,8 @@ en:
           attributes:
             stage:
               inclusion: "is not one of the allowed stage options"
+            name:
+              too_long: Must be 10 words or less
```

Comment:
I kinda hate "fewer" but pedants might squeal, although this looks like it's just the error message for devs rather than users

Reply:
That took me a second!